Update opensearch log integration (#1363)
aiven/devportal: docs/products/opensearch/howto/opensearch-log-integration.rst

Here are the steps needed to enable logs integration. This allows you to send your service's logs to your Aiven for OpenSearch® from another Aiven service.

1. On the **Services** page, click a service name.

2. On the **Logs** tab, click on **Enable logs integration**.

3. Select an existing OpenSearch instance or create a new one, then select **Continue**.
    - When creating a new service you will need to select the cloud, region and plan to use. You should also give your service a name. The service overview page shows the nodes rebuilding, and then indicates when they are ready.

>     - If you're already using OpenSearch on Aiven, you can use your running OpenSearch service as a destination for your metrics data. If you are a member of more than one Aiven projects with *operator* or *admin* access right, you need to choose the project first then your target OpenSearch service.

Reviewer: ```suggestion
    - If you're already using OpenSearch on Aiven, you can use your running OpenSearch service as a destination for your metrics data. If you are a member of more than one Aiven project with *operator* or *admin* access rights, you need to choose the project first then your target OpenSearch service.
```

You can change the configuration of the ``index prefix`` and ``index retention limit`` after the integration is enabled.

1. On the **Services** page, click a service name.
   
2. On the **Overview** page, scroll down to the **Service integrations**.

3. Click on **Manage integrations**.

4. On the **Enabled service integrations** list, find the service to configure.

> 5. Click on **Edit** button, and edit the parameters.

Reviewer: ```suggestion
5. Click **Edit**, and edit the parameters.
```

> 6. Once the parameters are chosen, click the **Edit** button.

Reviewer: ```suggestion
6. Once the parameters are chosen, click **Edit**.
```

In case, you are no longer interested in sending the logs from your service to OpenSearch, follow those steps to disable the integration:

1. On the **Overview** page, scroll down to the **Service integrations**.

2. Click on **Manage integrations**.

3. On the **Enabled service integrations** list, find the service to disable the integration.

> 4. Click the **Delete** button.

Reviewer: ```suggestion
4. Click **Delete**.
```
